Sources/AppDelegate.swift

```diff
@@ -14,6 +14,9 @@ import Swinject
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
+    var mainViewController: MainViewController?
```

Comment:
Wat doen we met dit soort optionals? Echt optional is hij niet maar word natuurlijk laat geinitialiseerd. Als we hem op meerdere plekken gebruiken is het wellicht wat onhandig om het een optional te laten zijn (wat doe je als hij op die plek nil is, mag immers niet voorkomen?).